showcase/src/app/demo-ui/components/card/card.component.ts

```diff
@@ -0,0 +1,47 @@
+import { Component, HostBinding, Input } from "@angular/core";
+
+@Component({
+	selector: "app-card",
+	templateUrl: "./card.component.html",
+	styleUrls: ["./card.component.scss"]
+})
+export class CardComponent {
```

Comment:
Maybe this component could be part of "reactive-forms-with-ngx-form-errors" folder as is this the case for "generic-search" 😊